src/MockClassifier.UnitTests/Extensions/MockHttpClientFactoryExtensions.cs

```diff
@@ -16,7 +16,7 @@ internal static class MockHttpClientFactoryExtensions
                 .Returns(() =>
                 {
                     var client = messageHandler.ToHttpClient();
-                    client.BaseAddress = new Uri("https://dmr.fakeurl.com");
+                    client.BaseAddress = new Uri("https://tablelogapi.azurewebsites.net/app/fromclassifier");
```

Comment:
Is this URL a real service you're using?

Reply:
Yes, it is, this is a service that Martin is hosting that keeps an inmemory log of calls being made to it and in turns makes callbacks also.  Good way of debugging behaviour.

Reply:
Yes, it is this which we can use until we have a better way of int testing: https://github.com/martinkearn/Table-Log-Api